jaxrs-spec/src/main/asciidoc/chapters/introduction/_status.adoc

```diff
@@ -28,3 +28,11 @@ The project team seeks feedback from the community on any aspect of this
 specification, please send comments to:
 
 jaxrs-dev@eclipse.org
+
+[[context-injection]]
+==== Support for @Context Injection
+
+As part of an effort to better align with https://jakarta.ee/specifications/cdi/[Jakarta CDI],
+future versions of this API will no longer support `@Context` injection and related types such as
+`ContextProvider`. As much as possible, all injection tasks will be delegated to
```

Comment:
Couple thoughts here:
- `ContextProvider` is not a Java type, so probably should be "context providers" or `ContextResolver`.
- I wonder if we really need to deprecate/remove context providers.  They can be pretty useful, and I think CDI would still support them - for example:
```
@Provider
MyMarkDownWriter implements MessageBodyWriter {
    @Inject //formerly @Context
    ContextResolver<MarkDownConfig> ctxResolver;

    //...
}
```

I'm not sure whether CDI is generic-aware to pull that off out of the box, but an extension should make it possible.

Reply:
Sorry, meant to say `ContextResolver`. I don't think we should duplicate any functionality in CDI, and I hope these can be replaced by CDI producers. In any case, the text is just providing _an example_ of other potential changes related to `@Context` removal, not a final decision.